src/main/java/com/google/cloud/tools/appengine/api/deploy/AppEngineArtifactStaging.java

```diff
@@ -22,7 +22,7 @@
  * Main interface for staging Java application for App Engine flexible environment before
  * deployment.
  */
-public interface AppEngineFlexibleStaging {
+public interface AppEngineArtifactStaging {
```

Comment:
artifact sounds like a Maven term, not an App Engine term. Maybe it makes sense here, but is there a more standard App Engine term for this thing? or even just rop it completely so `AppEngineStaging`? I'm not sure.